Framework/Geometry/src/Instrument/InstrumentVisitor.cpp

```diff
@@ -217,7 +216,6 @@ size_t InstrumentVisitor::registerDetector(const IDetector &detector) {
     (*m_componentIdToIndexMap)[detector.getComponentID()] = detectorIndex;
     (*m_componentIds)[detectorIndex] = detector.getComponentID();
     m_assemblySortedDetectorIndices->push_back(detectorIndex);
-    m_assemblySortedComponentIndices->push_back(detectorIndex);
```

Comment:
There should probably be a comment somewhere what the meaning of this is now. Maybe the doxygen in the header would be ok.